lib/stackInformation.js

```diff
@@ -171,11 +171,7 @@ module.exports = {
 			BbPromise.bind(this).then(this.aliasStackLoadAliasTemplates)
 		)
 		.spread((currentTemplate, aliasStackTemplates) => {
-			const currentAliasStackTemplate =
-				_.get(
-					_.first(_.remove(aliasStackTemplates, ['stack', `${this._provider.naming.getStackName()}-${this._alias}`])),
-					'template',
-					{});
+			const currentAliasStackTemplate = _.get(_.first(aliasStackTemplates, 'template', {}));
```

Comment:
I think this is wrong. The current template must be removed from the aliasStackTemplates, as was before.

Reply:
I tested this. the `_.remove()` function as it was used, didn't delete anything. I tried to delete it using _.without, but I ran into an error.

Reply:
`_.remove()` alters the aliasStackTemplates (removes the element) and returns the array of removed elements. This should have worked, because otherwise any alias resource differences would not have worked.

Reply:
sorry, a misunderstanding on my part. I think the latest push, should resolve this.